software/multiqc/main.nf

```diff
@@ -0,0 +1,32 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options = initOptions(params.options)
+
+process MULTIQC {
+    tag "multiqc"
```

Comment:
I think this should be fine since essentially `multiqc` is a single invocation for all pipelines. It's best if @drpatelh gives a 👍  for this as well, since generally `meta.id` is used for all other modules.